compositor_pipeline/src/pipeline/types.rs

```diff
@@ -125,6 +125,17 @@ pub(super) enum Samples {
     Stereo32Bit(Vec<(i32, i32)>),
 }
 
+impl Samples {
+    pub fn get_number_of_samples(&self) -> usize {
```

Comment:
NIT: getters in rust ususally are named without `get_`, I would use `sample_count` but `number_of_samples` is fine